src/euphorie/client/docx/compiler.py

```diff
@@ -832,9 +837,12 @@ def set_modules_rows(self, data):
                 count += 1
                 self.set_cell_risk(row_risk.cells[self.risk_description_col], risk)
                 if self.risk_answer_col is not None:
-                    row_risk.cells[self.risk_answer_col].text = (
-                        self.justifiable_map.get(answer) or ""
-                    )
+                    cell = row_risk.cells[self.risk_answer_col]
+                    cell.text = self.justifiable_map.get(answer) or ""
+                    font = self.justifiable_font.get(answer)
```

Comment:
I think this increases the complexity of the method, which is already complex. But ok...
Probably adding a separate method, expanded in the `DocxCompilerShort` would have been better.
But I am not able to really judge by just reading the PR.